src/settings.c

```diff
@@ -201,6 +201,9 @@ void check_and_correct_settings(struct settings *s) {
                 if (s->progress_bar_min_width > s->width.max) {
                         LOG_W("Progress bar min width is greater than the max width of the notification.");
                 }
+                if (s->progress_bar_corner_radius > 100) {
```

Comment:
The radius should be clamped to half the progress bar height, since that's the maximum that allows for a smooth circle.
This check currently also doesn't change the value itself.